pkg/server/container_create.go

```diff
@@ -155,7 +156,7 @@ func (c *criContainerdService) CreateContainer(ctx context.Context, r *runtime.C
 	return &runtime.CreateContainerResponse{ContainerId: id}, nil
 }
 
-func (c *criContainerdService) generateContainerSpec(id string, sandboxPid uint32, config *runtime.ContainerConfig,
+func (c *criContainerdService) generateContainerSpec(id string, sandbox sandboxstore.Sandbox, config *runtime.ContainerConfig,
```

Comment:
For now passing the whole struct because I dont see this to be heavy. I feel its a tradeoff between ever increasing arguements/object. WDYT?